src/app/src/App.jsx

```diff
@@ -78,6 +82,18 @@ class App extends Component {
                                     path={mainRoute}
                                     component={MapAndSidebar}
                                 />
+                                <Route
+                                    exact
+                                    path={claimFacilityRoute}
+                                    render={() => (
+                                        <FeatureFlag
+                                            flag={CLAIM_A_FACILITY}
+                                            alternative={<Route component={MapAndSidebar} />}
+                                        >
+                                            <Route component={ClaimFacility} />
```

Comment:
The wonky `Route` nesting here is because I wanted to continue getting the OAR ID from the path.